src/live_restore/live_restore_private.h

```diff
@@ -28,6 +28,8 @@
 #define WTI_OFFSET_TO_BIT(offset) (uint64_t)((offset) / (wt_off_t)lr_fh->allocsize)
 #define WTI_BIT_TO_OFFSET(bit) (wt_off_t)((bit)*lr_fh->allocsize)
 
+#define WTI_BITMAP_END(lr_fh) ((wt_off_t)(lr_fh)->allocsize * (wt_off_t)(lr_fh)->nbits)
```

Comment:
It feels like this macro should be with the other bitmap related ones, one line above. Then the blank line and then DEST_COMPLETE.

Reply:
b1545d03b3